libs/blocks/graybox/graybox.css

```diff
@@ -0,0 +1,283 @@
+:root {
+  --gb-container-bg: white;
+  --gb-outline-color: #1473E6;
+  --gb-container-btn-bg: #505050;
+  --gb-overlay-color: rgba(0 0 0 / 45%);
+  --gb-modal-bg: #323232;
+  --gb-container-border: 4px solid var(--gb-outline-color);
+  --gb-page-outline-border: 8px solid var(--gb-outline-color);
+  --gb-toggle-arrow: 3px solid white;
+}
+
+.graybox {
+  display: none;
+}
+
+.gb-graybox-body {
+  position: relative;
+}
+
+.gb-graybox-body:not(.gb-no-border)::before {
+  content: '';
+  position: absolute;
+  top: calc(50% + 8px);
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: calc(100% - 15.5px);
```

Comment:
These values seem a little magic... are we safe they wont change in the future or should we read them from a general variable?
50% + 8px, 100% - 15.5px 🤔

Reply:
This psuedoelement is what creates the blue border around the entire page.  The top and width calcs are to account for the width of the blue border.